tests/product/test_status.py

```diff
@@ -48,11 +46,13 @@ def test_status_happy_path(self):
 
         # Test with worker not started
```

Comment:
In a next patch could you move the tests that aren't actually happy path tests into their own methods?  It would make it easier to quickly diagnose jenkins issues.